src/js/components/modals/AppModalComponent.jsx

```diff
@@ -61,28 +61,19 @@ var AppModalComponent = React.createClass({
     Mousetrap.bind(["command+enter", "ctrl+enter"], () => this.handleSubmit());
 
     if (!this.hasInfoObject()) {
-      InfoStore.on(InfoEvents.CHANGE, this.onInfoChange);
-      InfoStore.on(InfoEvents.REQUEST_ERROR, this.onInfoChange);
+      InfoStore.once(InfoEvents.CHANGE, this.onInfoChange);
```

Comment:
I think these listeners will be never removed now, because no `removeListener` is called anymore..
But maybe this is done automatically, not sure?